config/model-game.js

```diff
@@ -0,0 +1,30 @@
+const Sequelize = require('sequelize');
+const sequelize = require('./configure.js')
```

Comment:
find a better name for this variable, it could be easily confused with the other one "Sequelize"

Reply:
fixed errors